docs/development-environment.md

```diff
@@ -167,34 +167,34 @@ There are three types of builds:
 	The standard development build will create un-minified versions of the JavaScript and CSS files. To build Jetpack like this run:
 	
 	```sh
-	yarn build
+	pnpm build
 	```
 	
 * ### Continuous Development build
-	By default the development build above will run once and if you change any of the files, you need to run `yarn build` again to see the changes on the site. If you want to avoid that, you can run a continuous build that will rebuild anytime it sees any changes on your local filesystem. To run it, use:
+	By default the development build above will run once and if you change any of the files, you need to run `pnpm build` again to see the changes on the site. If you want to avoid that, you can run a continuous build that will rebuild anytime it sees any changes on your local filesystem. To run it, use:
 	
 	```sh
-	yarn watch
+	pnpm watch
 	```	
 
 * ### Production build
 	The production build will generate minified files without duplicated code (resulting from dependencies) and will also generate the matching source map and language files. To build it use:
 	
 	```sh
-	yarn build-production-client
+	pnpm build-production-client
 	```
 
 ### A note on building Jetpack and Node.js versions
 
-We try to frequently keep the Node version we use up to date. So, eventually you may need to refresh your package dependencies (i.e., the `node_modules` directories). This is because some dependencies are built specifically for the Node version you used when you installed them (either by running `yarn build` or `yarn`).
+We try to frequently keep the Node version we use up to date. So, eventually you may need to refresh your package dependencies (i.e., the `node_modules` directories). This is because some dependencies are built specifically for the Node version you used when you installed them (either by running `pnpm build` or `pnpm install`).
 
 We recommend usage of [nvm](https://github.com/nvm-sh/nvm/) for managing different Node versions on the same environment.
 
-**Note:** If you have previously run the Jetpack build tasks (e.g. `yarn build`), and didn't come back to it for a long time, you can
+**Note:** If you have previously run the Jetpack build tasks (e.g. `pnpm build`), and didn't come back to it for a long time, you can
 run this command before building again. Otherwise you may experience errors on the command line while trying to build.
 
 ```sh
-yarn distclean
+pnpm distclean
```

Comment:
This is fine for now I think. It mentions it is deprecated for me, but we can wait until the CLI clean command is ready before updating the docs.

Reply:
Yeah, as I noted in the commit message on eb625dfdc7 updating all the outdated docs seems out of scope for this PR.